packages/explat/src/assignment.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { stringify } from 'qs';
+
+/**
+ * Internal dependencies
+ */
+import { logError } from './error';
+
+const EXPLAT_VERSION = '0.1.0';
+
+export const fetchExperimentAssignment = async ( {
+	experimentName,
+	anonId,
+}: {
+	experimentName: string;
+	anonId: string | null;
+} ): Promise< unknown > => {
+	if ( ! window.wcTracks.isEnabled ) {
+		throw new Error(
+			`Tracking is disabled, can't fetch experimentAssignment`
+		);
+	}
+
+	const params = stringify( {
+		experiment_name: experimentName,
+		anon_id: anonId ?? undefined,
+	} );
+
+	try {
```

Comment:
No need to wrap this in a try-catch block

Reply:
removed in https://github.com/woocommerce/woocommerce-admin/pull/6669/commits/f59c92ea5b720f0a7c5bf2f747d3af6c242c5396